README.md

```diff
@@ -60,10 +59,10 @@ Use "optimus [command] --help" for more information about a command.
 ## Documentation
 Explore the following resources to get started with Optimus:
 
-* [Guides](https://odpf.github.io/optimus/guides/create-job/) provides guidance on using Optimus.
```

Comment:
Let's keep these to provide an overview context of these links. Feel free to change the language to provide more context though.